<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MJpeg Viewer</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="IP_label">
              <property name="text">
               <string>IP:</string>
              </property>
              <property name="buddy">
               <cstring>IP_line_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="IP_line_edit">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>10.0.250.150</string>
              </property>
              <property name="maxLength">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="port_label">
              <property name="text">
               <string>Port:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="port_spin_box">
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>1100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ratio_label">
              <property name="text">
               <string>Ratio:</string>
              </property>
              <property name="buddy">
               <cstring>ratio_combo_box</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ratio_combo_box">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>IgnoreAspectRatio</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>KeepAspectRatio</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>KeepAspectRatioByExpanding</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="request_label">
              <property name="text">
               <string>Request:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="request_edit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>GET /?action=stream HTTP/1.1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="fps_frames_label">
              <property name="toolTip">
               <string>The number of frames through which the FPS will be recalculated</string>
              </property>
              <property name="text">
               <string>FPS_frames:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="fps_frames_spin_box">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>The number of frames through which the FPS will be recalculated</string>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="connect_button">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="mjpeg_label">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string>MJpeg</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>connect()</slot>
 </slots>
</ui>
